gremlin-python/src/main/python/example/example.py

```diff
@@ -0,0 +1,157 @@
+
+
+def main():
+    connection_example()
+    basic_gremlin_example()
+    modern_traversal_example()
+
+
+def connection_example():
+    # connect to a remote server that is compatible with the Gremlin Server protocol. for those who
+    # downloaded and are using Gremlin Server directly be sure that it is running locally with:
+    #
+    # bin/gremlin-server.sh console
+    #
+    # which starts it in "console" mode with an empty in-memory TinkerGraph ready to go bound to a
+    # variable named "g" as referenced in the following line.
+    g = traversal().with_remote(DriverRemoteConnection('ws://localhost:8182/gremlin', 'g'))
+
+    # connecting with plain text and Kerberos SASL authentication
+    rc = DriverRemoteConnection('ws://localhost:8182/gremlin', 'g', username='stephen', password='password')
+    g = traversal().with_remote(rc)
+    rc.close()
+
+    rc = DriverRemoteConnection('ws://localhost:8182/gremlin', 'g', kerberized_service='gremlin@hostname.your.org')
+    g = traversal().with_remote(rc)
+    rc.close()
+
+    # connecting with customized configurations
+    rc = DriverRemoteConnection(
+        'ws://localhost:8182/gremlin', 'g',
+        username="", password="", kerberized_service='',
+        message_serializer=GraphBinarySerializersV1(), graphson_reader=None,
+        graphson_writer=None, headers=None, session=None,
+        enable_user_agent_on_connect=True
+    )
+    g = traversal().with_remote(rc)
+    rc.close()
+
+
+def basic_gremlin_example():
+    g = traversal().with_remote(DriverRemoteConnection('ws://localhost:8182/gremlin', 'g'))
+
+    # basic Gremlin: adding and retrieving data
+    v1 = g.add_v('person').property('name', 'marko').next()
+    v2 = g.add_v('person').property('name', 'stephen').next()
+    v3 = g.add_v('person').property('name', 'vadas').next()
+
+    # be sure to use a terminating step like next() or iterate() so that the traversal "executes"
+    # iterate() does not return any data and is used to just generate side-effects (i.e. write data to the database)
+    g.V(v1).add_e('knows').to(v2).property('weight', 0.75).iterate()
+    g.V(v1).add_e('knows').to(v3).property('weight', 0.75).iterate()
+
+    # retrieve the data from the "marko" vertex
+    marko = g.V().has('person', 'name', 'marko').values('name').next()
+    print("name: " + marko)
+
+    # find the "marko" vertex and then traverse to the people he "knows" and return their data
+    people_marko_knows = g.V().has('person', 'name', 'marko').out('knows').values('name').toList()
+    for person in people_marko_knows:
+        print("marko knows " + person)
+
+
+def modern_traversal_example():
+    # This example requires the Modern toy graph to be preloaded upon launching the Gremlin server.
+    # For details, see https://tinkerpop.apache.org/docs/current/reference/#gremlin-server-docker-image and use
+    # conf/gremlin-server-modern.yaml.
+    rc = DriverRemoteConnection('ws://localhost:8182/gremlin', 'g')
```

Comment:
```suggestion
    rc = DriverRemoteConnection('ws://localhost:8182/gremlin', 'gmodern')
```

Reply:
Kept as is since current connection is able to load the Modern graph given that the user follows instructions in the ref link provided